examples/jsm/renderers/webgpu/WebGPUBindings.js

```diff
@@ -155,39 +155,31 @@ class WebGPUBindings {
 			} else if ( binding.isSampler ) {
 
 				const material = object.material;
-				const texture = material[ binding.name ];
+				const texture = binding.texture;
 
-				if ( texture !== null ) {
```

Comment:
If we consider the texture parameter of `WebGPUSampledTexture` as mandatory, then removing this check is okay. Meaning `WebGPUSampledTexture.texture` can never be `null`.